go/chat/types/interfaces.go

```diff
@@ -618,7 +618,7 @@ type EmojiSource interface {
 		newAlias, existingAlias string) (chat1.EmojiRemoteSource, error)
 	Remove(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID, alias string) error
 	Get(ctx context.Context, uid gregor1.UID, convID *chat1.ConversationID, opts chat1.EmojiFetchOpts) (chat1.UserEmojis, error)
-	Decorate(ctx context.Context, body string, convID chat1.ConversationID, emojis []chat1.HarvestedEmoji) string
+	Decorate(ctx context.Context, body string, convID chat1.ConversationID, emojis []chat1.HarvestedEmoji, parseBig bool) string
```

Comment:
Can we change this to take the message type as a param, and make the decision about bigness in `Decorate` as opposed to the caller? I think that feels a little better, and might be more flexible in the future. Also helps avoid a `bool` param which people can sometimes find confusing.

Reply:
ah yep, makes sense